shared/chat/inbox/index.desktop.tsx

```diff
@@ -227,7 +212,22 @@ class Inbox extends React.Component<T.Props, State> {
       <ErrorBoundary>
         <div style={styles.container}>
           <div style={styles.list}>
-            <AutoSizer>{this.sizeChild}</AutoSizer>
+            <AutoSizer>
```

Comment:
i  moved this back as it was incorrect to hoist it. Since it depends on props, if i move it into a property on a render it won't notice a change as the closure is stable. Another option would be to memoize it based off of this.props.rows or something but we don't really gain anything by doing that so just making it inline again

Reply:
This is a little tricky, a comment by `listChild` saying it works because it doesn't depend on props would be good.